src/megaapi_impl.cpp

```diff
@@ -615,7 +623,10 @@ MegaNodePrivate *MegaNodePrivate::unserialize(string *d)
     char hasChatAuth = MemAccess::get<char>(ptr);
     ptr += sizeof(char);
 
-    if (memcmp(ptr, "\0\0\0\0\0\0", 7))
+    char hasOwner = MemAccess::get<handle>(ptr);
+    ptr += sizeof(char);
+
+    if (memcmp(ptr, "\0\0\0\0\0", 6))
```

Comment:
following `ptr += 7` shouldn't be `ptr += 6` ?

Reply:
Totally agree. Good catch, thanks